nilearn/input_data/nifti_masker.py

```diff
@@ -186,7 +182,7 @@ def fit(self, imgs=None, y=None):
         # Resampling: allows the user to change the affine, the shape or both
         if self.verbose > 0:
             print("[%s.fit] Resampling mask" % self.__class__.__name__)
-        self.mask_img_ = self._cache(image.resample_img, func_memory_level=1)(
+        self.mask_img_ = self._cache(image.resample_img, 1)(
```

Comment:
In general, what is the strategy to set memory_level / func_memory elvel to 1 or 2 ? Should there be a pragraph on it in the doc ?